hooks/pre-command

```diff
@@ -40,16 +63,21 @@ maybe_download_sc() {
   case "${OSTYPE}" in
     linux*)
       os="linux"
-      # TODO: cache this download on the agent
-      curl -fsSL --retry 5 "https://saucelabs.com/downloads/sc-${sauce_connect_version}-${os}.tar.gz" | tar -xzf - "sc-${sauce_connect_version}-${os}/bin/sc"
+
+      local sc_tarball_path
+      sc_tarball_path=$(maybe_download_sc_archive "${sauce_connect_version}" "${os}")
+
+      tar -xzf "${sc_tarball_path}" "sc-${sauce_connect_version}-${os}/bin/sc"
```

Comment:
This may be incorrect – not sure if the behaviour of `tar` changes when it's not accepting from `stdin`

Reply:
It should be fine, but usually it's called without the `-` in front of the single letter options, like
```suggestion
      tar xzf "${sc_tarball_path}" "sc-${sauce_connect_version}-${os}/bin/sc"
```

Reply:
232cfdfbad455efc84ac221ce0265ccea67af571